app/views/layouts/_footer.html.erb

```diff
@@ -1,34 +1,55 @@
-<div id="footer">
-  <div class="container">
-    <div>
-      <b>BiteSite Inc.</b><br />
-      Custom Software<br />
-      info@bitesite.ca<br />
-      204 - 78 George St.<br />
-      Ottawa, ON<br />
-      K1N 5W1<br />
-    </div>
-
-    <div>
+<%= javascript_pack_tag "footer" %> 
+
+
+<div id="footer-component-mount-point">
+
+  <div class="container fgs-al  fgs-al-align-items-flex-start  fgs-al-justify-content-space-between">
       
-    </div>
+      <div class="company-info fgs-ali fgs-al fgs-al-v fgs-al-g-10 ">
+      <span class="title fgs-ali">2021 BiteSite Inc.</span>
+        
+        <div class="fgs-ali fgs-al fgs-al-v fgs-al-g-6">
+          <span class="info fgs-ali">204 - 78 George St.</span>
+          <span class="info fgs-ali">Ottawa, ON</span>
+          <span class="info fgs-ali">K1N 5W1</span>
+          <span class="info fgs-ali">info@bitesite.ca</span>
 
-    <div>
-      <div class='social-links'>
-        <%= link_to fa_icon('linkedin'), 'https://www.linkedin.com/company/2894614/' %>
-        <%= link_to fa_icon('twitter'), 'https://www.twitter.com/bitesite' %>
-        <%= link_to fa_icon('youtube-play'), 'https://www.youtube.com/user/bitesiteca' %>
+        </div>
       </div>
+  <div>
+
+   
+      <div class='links fgs-ali fgs-al fgs-al-h fgs-al-align-items-flex-start'>
+
+          <div class="company-links fgs-ali fgs-al fgs-al-v">
+          <span class="links-title fgs-ali">COMPANY</span>
+          <%= link_to "Services", services_path, :class => "fgs-ali" %>
+          <%= link_to "About", about_path, :class => "fgs-ali" %>
+          <%= link_to "Careers", careers_path, :class => "fgs-ali" %>
+          <%= link_to "Contact Us", contact_path, :class => "fgs-ali" %>
+          </div>
```

Comment:
@NRaeewal I feel a lot of code in the erb file you transferred to footer.jsx. I don't think you need it here.